documentation/Installing-Clear-Containers-on-Fedora-25.md

```diff
@@ -0,0 +1,63 @@
+#Install the Clear Containers runtime on Fedora 25
+
+##Install the Clear Containers runtime
+```
+$ sudo -E dnf config-manager --add-repo
+http://download.opensuse.org/repositories/home:clearlinux:preview:clear-containers-2.1/Fedora\_25/home:clearlinux:preview:clear-containers-2.1.repo
+$ sudo dnf install cc-oci-runtime linux-container
+```
+
+##Install docker 1.12
+```
+$ sudo dnf config-manager --add-repo \
+	https://docs.docker.com/engine/installation/linux/repo_files/fedora/docker.repo
+$ sudo dnf install docker-engine-1.12.1-1.fc25
```

Comment:
Just to check @egernst - is this 1.12.1 for fc25 still valid? - I had a browse of the repo and did not see it - and #687 implies that maybe it is not... I'm not set up to easily try this - anybody else verify??